Sources/Core/iOS/NetworkObserver.swift

```diff
@@ -82,19 +82,19 @@ private class NetworkIndicatorController {
     // Public API
 
     func networkActivityDidStart() {
-        assert(Thread.isMainThread(), "Altering network activity indicator state can only be done on the main thread.")
+        assert(Thread.isMainThread, "Altering network activity indicator state can only be done on the main thread.")
```

Comment:
I will rebase these changes on-top-of latest `develop` as this has now been improved. We have `Queue.isMainQueue` which doesn't use `Thread` class property anymore.